src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFandom.java

```diff
@@ -0,0 +1,100 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.templates.fandom.AdsFandomTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+
+public class TestAdsFandom extends AdsFandomTestTemplate {
+
+  private void getJquery() {
```

Comment:
Instead of `document.querySelector` it's better to use Selenium API to find the web element.